packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -479,6 +479,9 @@ def _download_bundle_from_master(dcos_api_session, master_index, bundle):
         'var/lib/dcos/exhibitor/zookeeper/snapshot/myid.gz',
         'var/lib/dcos/exhibitor/conf/zoo.cfg.gz',
         'var/lib/dcos/mesos/log/mesos-master.log.gz',
+        'var/lib/dcos/mesos/log/mesos-master.log.1.gz',
```

Comment:
Seems like ether the dcos-config.yaml:1772 needs to change, or this should drop the `.gz`.

Reply:
dcos-diagnostics compress file content so the bundled file ends with `.gz` suffix that's why we have `.gz.gz`  below